ocs_ci/utility/utils.py

```diff
@@ -1813,6 +1813,19 @@ def skipif_ocs_version(expressions):
     return skip_this
 
 
+def skipif_lso_deployment():
+    """
+    Check if the deployment is LSO based before starting add_capacity test
+
+    """
+    from ocs_ci.ocs.resources import csv
```

Comment:
please do the import in the file beginning

Reply:
If we import at the beginning import is failing. Seems something related to circular imports.